modules/CONT-02_Using-branches.yml

```diff
@@ -26,10 +26,10 @@ screens:
         - do: Go to another repo and then return to the class repo
           say: "However, a word of caution. When you leave the repository and come back, notice that GitHub automatically assumes you want to see the items on the master branch."
         - do: Click the branch dropdown and reselect the branch
-          say: "If you want to continue working on your branch, you will need to reselect it using the branch dropdown."
+          say: "If you want to continue working on your branch, you will need to reselect it using the branch dropdown. On the next page you will create your own branch."
```

Comment:
Not sure of how everything will be rendered in Certify, but is there a difference between a page and a section?  If not, I wonder if it would be better to use "section" for everything.  

Since these videos may eventually have more than one channel of distribution, I think it would be good to tweak any language related tied to navigation within a particular interface.